resource-service/README.md

```diff
@@ -18,46 +18,81 @@ limitations under the License.
 
 # Resource Service
 
-The Resource service accepts an incoming message from the `user` Kafka topic which contains the resourceId that is being accessed 
-(this could be an actual file, or a directory that could contain many files and/or sub-directories). The service will then query the backing store to
-see if the requested resourceId has been stored. If this is not the case then the request will be passed onto the local implementation of 
-the Resource service. All the returned resources will be within an Akka stream, each element in the stream is then consumed and
-added to the `resource` Kafka topic to be processed by the Policy service.
+> :warning:
+Windows users will have problems with Hadoop integration testing. 
+Included [here](src/component-tests/resources/hadoop-3.2.1/bin) is a Windows-compatible set of hadoop binaries. 
+To 'install' on Windows, copy the [hadoop.dll](src/component-tests/resources/hadoop-3.2.1/bin/hadoop.dll) to `C:\Windows\System32`. 
+This should enable the `HadoopResourceServiceTest` to run correctly. 
 
-## Message Model and Database Domain
+At it's core, the Resource Service is an implementation of the `ResourceService` interface:
+```java
+interface ResourceService {
+    Iterator<LeafResource> getResourcesById(final String resourceId);
```

Comment:
The concept of LeafResource could do with an explanation.  The readme would be a good place to explain this and how it fits into the bigger picture.

Reply:
Done